pkg/cmd/pipeline/logs_test.go

```diff
@@ -648,264 +303,294 @@ func TestLogs_interactive_ask_all_last_run(t *testing.T) {
+						return err
+					}
+
+					if _, err := c.ExpectString("Select pipelinerun:"); err != nil {
+						return err
+					}
+
+					if _, err := c.SendLine(string(terminal.KeyArrowDown)); err != nil {
+						return err
+					}
+
+					if _, err := c.ExpectString(prName2 + " started 3 minutes ago"); err != nil {
+						return err
+					}
+
+					if _, err := c.SendLine(string(terminal.KeyArrowDown)); err != nil {
+						return err
+					}
+
+					if _, err := c.ExpectString(prName + " started 5 minutes ago"); err != nil {
+						return err
+					}
+
+					if _, err := c.SendLine(string(terminal.KeyEnter)); err != nil {
+						return err
+					}
+
+					return nil
+				},
 			},
 		},
-	}
-	opts := logOpts(prName, ns, 5, true, cs)
-	for _, test := range tests {
-		t.Run(test.name, func(t *testing.T) {
-			RunPromptTest(t, test, opts)
-		})
-	}
-}
-
-func TestLogs_last_run_diff_namespace(t *testing.T) {
-	clock := clockwork.NewFakeClock()
-
-	cs, _ := test.SeedTestData(t, pipelinetest.Data{
-		Pipelines: []*v1alpha1.Pipeline{
-			tb.Pipeline(pipelineName, ns,
-				// created  15 minutes back
-				cb.PipelineCreationTimestamp(clock.Now().Add(-15*time.Minute)),
-			),
-		},
-		PipelineRuns: []*v1alpha1.PipelineRun{
-
-			tb.PipelineRun(prName, ns,
-				cb.PipelineRunCreationTimestamp(clock.Now().Add(-10*time.Minute)),
-				tb.PipelineRunLabel("tekton.dev/pipeline", pipelineName),
-				tb.PipelineRunSpec(pipelineName),
-				tb.PipelineRunStatus(
-					tb.PipelineRunStatusCondition(apis.Condition{
-						Status: corev1.ConditionTrue,
-						Reason: resources.ReasonSucceeded,
-					}),
-					// pipeline run started 5 minutes ago
-					tb.PipelineRunStartTime(clock.Now().Add(-5*time.Minute)),
-					// takes 10 minutes to complete
-					cb.PipelineRunCompletionTime(clock.Now().Add(10*time.Minute)),
-				),
-			),
-			tb.PipelineRun(prName2, ns,
-				cb.PipelineRunCreationTimestamp(clock.Now().Add(-8*time.Minute)),
-				tb.PipelineRunLabel("tekton.dev/pipeline", pipelineName),
-				tb.PipelineRunSpec(pipelineName),
-				tb.PipelineRunStatus(
-					tb.PipelineRunStatusCondition(apis.Condition{
-						Status: corev1.ConditionTrue,
-						Reason: resources.ReasonSucceeded,
-					}),
-					// pipeline run started 3 minutes ago
-					tb.PipelineRunStartTime(clock.Now().Add(-3*time.Minute)),
-					// takes 10 minutes to complete
-					cb.PipelineRunCompletionTime(clock.Now().Add(10*time.Minute)),
-				),
-			),
-		},
-		Namespaces: []*corev1.Namespace{
-			{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: ns,
+		{
+			name:      "basic interaction4",
+			limit:     1,
+			last:      false,
+			namespace: ns,
+			input:     cs,
+			prompt: htest.PromptTest{
+				CmdArgs: []string{pipelineName},
+				Procedure: func(c *goexpect.Console) error {
+					if _, err := c.ExpectString("output-pipeline"); err != nil {
+						return err
+					}
+
+					if _, err := c.SendLine(string(terminal.KeyEnter)); err != nil {
+						return err
+					}
```

Comment:
Why in this case are we expecting a pipeline prompt if the pipeline name is included as part of the command?